util/test/sub_test

```diff
@@ -1032,6 +1033,26 @@ for testname in testsrc:
             if do_not_test:
                 break
 
+        elif (suffix=='.suppressif' and (os.access(f, os.R_OK))):
+            testskipiffile=True
+            skiptest=runSkipIf(f)
+            try:
+                skipme=False
```

Comment:
Could we call this something other than "skipme" since suppressed tests aren't skipped (or at least, are not supposed to be), but are supposed to be run and verified that they failed as expected?